acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +82,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
-          step "assert no redacted data in syslog" do
-            #sigh.  gee, thanks for the help, beaker!
-            result = dump_puppet_log(agent)
```

Comment:
yeah, been meaning to move away from that travesty of a method.  although i guess i can't complain because its _comment_ says it is private.  :hurtrealbad: that we're duplicating the syslog location info all over the place, but this can be fixed in beaker later